addons/web_editor/static/src/components/media_dialog/search_media.js

```diff
@@ -3,19 +3,28 @@
 import { debounce } from '@web/core/utils/timing';
 import { useAutofocus } from '@web/core/utils/hooks';
 
-import { Component, onWillUnmount, xml } from "@odoo/owl";
+import { Component, onWillUnmount, xml, useEffect, useState } from "@odoo/owl";
 
 export class SearchMedia extends Component {
     setup() {
         useAutofocus();
-        this.search = debounce((ev) => this.props.search(ev.target.value), 1000);
+        this.debouncedSearch = debounce(this.props.search, 1000);
+
+        this.state = useState({
+            input: '',
+        });
+
         onWillUnmount(() => {
             this.search.cancel();
         });
```

Comment:
What is `this.search` now that you removed it on the line just above this ? :thinking:

Reply:
Just pushed to fix that